app/main.cpp

```diff
@@ -511,12 +511,11 @@ int main( int argc, char *argv[] )
 #endif
   engine.rootContext()->setContextProperty( "__version", version );
 
-  // Enabling HighDPI scaling attribute (at the beggining of the main function) removes the
-  // need for manually calculating dp factor and multiplying it with each pixel value in qml.
-  // However, we keep the multiplications in place (right now we multiply only with 1),
-  // in case we would encounter a screen that cannot be scaled automatically.
+  // Even though enabling QT's HighDPI scaling removes the need to multiply pixel values with dp,
+  // there are screens that need a "little help", because system DPR has different value than the
+  // one we calculated. In these scenarios we use a ratio between real (our) DPR and DPR reported by QT.
   // Use `value * __dp` for each pixel value in QML
-  engine.rootContext()->setContextProperty( "__dp", 1 );
+  engine.rootContext()->setContextProperty( "__dp", InputUtils::calculateDpRatio() );
```

Comment:
I am wondering if this sets __dp correctly - probably there are no top-level windows yet, so it just uses 1 ? because on my phone it has still the same increased scale of the whole GUI

Reply:
Correct again